src/utils/metadata/descriptions.json

```diff
@@ -1,45 +1,45 @@
+  "tooltip_datafarming_3_round_asset_APY": "Data Farming Rewards APY for the last 3 rounds.",
+  "tooltip_datafarming_last_round_asset_APY": "Data Farming Last Round Per Asset Rewards APY resets each Round and it is calculated using rewards compounding.\n\nAPY = ((1 + lastRoundYield)^52) - 1",
+  "tooltip_datafarming_last_round_asset_APR": "Data Farming Last Round Per Asset Rewards APR resets each Round and it is relative to veOCEAN allocated and consume volume.\n\nAPR = lastRoundYield * 52",
+  "tooltip_datafarming_last_round_asset_yield": "Data Farming Last Round Per Asset Rewards Yield represents the ROI for the round.\n\nlastRoundYield = rewards Amount / ocean allocated",
+  "tooltip_datafarming_current_round_asset_APY": "Data Farming Current Round Per Asset Rewards APY resets each Round and it is calculated using rewards compounding.\n\nAPY = ((1 + roundYield)^52) - 1",
+  "tooltip_datafarming_current_round_asset_APR": "Data Farming Current Round Per Asset Rewards APR resets each Round and it is relative to veOCEAN allocated and consume volume.\n\nAPR = roundYield * 52",
+  "tooltip_datafarming_current_round_asset_yield": "Data Farming Current Round Per Asset Rewards Yield represents the ROI for the round.\n\nroundYield = rewards Amount / ocean allocated",
+  "tooltip_datafarming_my_ve_allocation": "This number represents the total amount of your veOCEAN allocated towards each dataset.",
+  "tooltip_rewards_history_rounds": "Data Farming rewards distribution rounds. Rounds happen every week starting and ending each Thursday.",
+  "tooltip_rewards_overview": "Data Farming APY for both Passive & Active Rewards.\n\nRounds happen every week starting and <a href=https://docs.oceanprotocol.com/veocean-data-farming/df-intro#when-to-claim target=_blank>ending each Thursday at midnight GMT.</a>",
+  "tooltip_rewards_apy_passive": "Data Farming Passive Rewards APY is calculated relative to your Ocean locked.\n\nThe longer you lock, the higher your APY will be.",
+  "tooltip_rewards_apy_active": "Data Farming Active Rewards APY resets each Round and is relative to your Ocean locked and DataNFT curation.\n\nThe longer you lock, and the better you curate, the higher your APY will be.",
+  "tooltip_rewards_history_active_rewards": "Rewards distributed for active curation of datasets. Active rewards are emitted based on Data Consume Volume.<br/><br/>Please check the docs for more info.",
+  "tooltip_rewards_apy_passive_history": "Average / User\n\nHistorical Passive Rewards distributed to users holding veoCEAN.",
+  "tooltip_rewards_apy_volume_history": "Average / User\n\nHistorical Volume DF Rewards distributed to users engaged in Data Farming.",
+  "tooltip_active_rewards": "The formula for calculating the active rewards can change. You can find the formula for the current round <a href=https://docs.oceanprotocol.com/veocean-data-farming-tutorial target=_blank>here</a>"
+}
```

Comment:
Our linters are conflicting.
Lots of changes happening although very little changed.

Reply:
Yeah, should setup a linter config file, maybe once WC PR it's merged because that contains a new module bundler